### PR TITLE
Created snake task

### PR DESCRIPTION
**Pull request by olya-yevsieienko**
- [DEMO LINK](https://olya-yevsieienko.github.io/layout_snake/)
- [TEST REPORT LINK](https://olya-yevsieienko.github.io/layout_snake/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  .block--5 {
+    order: 3;
+  }
+
+  .block--6 {
+    order: 4;
+  }
+}
+
+@media (max-width: 599px) {
```
Comment (jstmpelowycz) [inline src/style.css:75]: That is more obvious to create a media query this way
```suggestion
@media (min-width: 600px) {
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  background-color: rgb(51, 0, 0);
+  order: 2;
+}
+
+.block--6 {
+  background-color: rgb(0, 0, 0);
+  order: 1;
+}
+
+@media (max-width: 899px) {
```
Comment (jstmpelowycz) [inline src/style.css:53]: That is more obvious to create a media query this way
```suggestion
@media (min-width: 900px) {
```

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-grow: 1;
+  flex-basis: 33.3%;
```
Comment (jstmpelowycz) [inline src/style.css:21]:
```suggestion
  flex-basis: calc(100% / 3);
```

**Review by jstmpelowycz — CHANGES_REQUESTED** [review]
A few things left to fix, and you're done

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
```
Comment (jstmpelowycz) [inline src/style.css:16]: The width of each block should be not less than its height (300px)

Comment (jstmpelowycz) [inline src/style.css:16]: The width of each block should be not less than its height (300px)

**Review by jstmpelowycz — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 4 above)
Comment (jstmpelowycz) [reply src/style.css:15]: Add `min-width`

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+  .block--4 {
+    order: 2;
+  }
+
+  .block--5 {
+    order: 1;
+  }
+}
+
+@media (min-width: 600px) and (max-width: 899px) {
```
Comment (jstmpelowycz) [inline src/style.css:64]: Do you need `max-width`?

**Review by jstmpelowycz — CHANGES_REQUESTED** [review]
Small hints left

**Comment by olya-yevsieienko** [comment]
I have a question, if I set flex-basic for each block, do I still need to
write min-width?

пт, 13 мая 2022 г. в 10:31, Dmytro Pelovych ***@***.***>:

**Comment by olya-yevsieienko** [comment]
Okay, thanks! I fixed it.

ср, 11 мая 2022 г. в 20:14, Dmytro Pelovych ***@***.***>:

**Pull request closed without merging**